Changed the bill-to-account field to look like multiple select buttons

tejaswini-kesireddy opened the pull request.

Commit:
Changed the bill-to-account field to look like multiple select buttons

Commit:
Changed radio button to integer buttons

shaneallgeier committed:
Render labels for new charge_amount field

shaneallgeier committed:
Actually include missing html template for charge_amount widget

Commit:
Fixed the total charge amount issue

Commit:
Changed quantity and total amount labels

Commit:
Removed Monroe Mailbox Key field.

Commit:
Deleted editor files

shaneallgeier commented on keyformproject/Scripts/activate:
I'm not entirely sure what created this "Scripts" directory but it should probably be excluded/deleted as well. Feel free to add "Scripts/" into the .gitignore

shaneallgeier commented on keyformproject/pyvenv.cfg:
I guess we should also ignore pyvenv.cfg files

shaneallgeier commented on manage.py:
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+# 1.11.29#!/usr/bin/env python
```
Not sure what happened here, but I'd like this change to be reverted back to what it was.

shaneallgeier commented on keyform/forms.py:
```diff
@@ -1,25 +1,100 @@
+from random import choices
```
This line should be deleted since it's an unused import

shaneallgeier commented on keyform/models.py:
Since we technically made changes to the models (even though they're purely cosmetic), we need to run `uv run manage.py makemigrations keyform` to create a new database migration to update the database state.

Commit:
Deleted editor files
cleaned unused imports

Commit:
Edited manage.py

Commit:
Add docstrings

shaneallgeier commented on keyform/forms.py:
```diff
@@ -1,25 +1,98 @@
+        return total_amt
 
 class CreateForm(forms.ModelForm):
+    billing_items = [
```
This `dict` is no longer needed since it's unused

shaneallgeier commented:
Some new changes requested:

- Using client-side javascript, prevent people from typing negative numbers into the textboxes (it's already blocked on the Python side)
- Make a new permission to allow certain users to charge $0
  - First off, do we even currently reject $0? We need to add that if it's not already there.
  - For the permission checking stuff, we can check [`request.user.has_perm('keyform.can_charge_zero')`](https://devdocs.io/django~1.11/ref/contrib/auth#django.contrib.auth.models.User.has_perm) in `KeyRequest.get_form()` and then pass it pass it in as `can_charge_zero` boolean kwarg to `CreateForm`. And then, in `CreateForm`'s `__init__()` function, store it in `self.can_charge_zero`. Then we should be able to access it and check against it in `clean()`

Commit:
Update js to prevent negative numbers
Add permission to allow certain users to charge 0
Create a new database migration to update the database state.
